### ISSUE TITLE
New feature : Checking if website is down 

### ISSUE THREAD

**Issue by rzafiamy** [bernaferrari/ChangeDetection#47]
Nice app ! 
Is it possible to add this behaviour ? I manage several websites and want to check regularly the state of my websites. 
I am developer also , I may give some help if needed....

**Comment by bernaferrari** [comment]
Yeah, it is. I'm not currently maintaining this app 😩 I need to redesign to Compose but Compose is not yet stable. I think you'll be happier with my competitor-friend: https://github.com/afollestad/nock-nock

his app was made directly with your use case in mind

**Comment by rzafiamy** [comment]
Thanks , I'have tested it and  indeed Nock Nock fills up my request... Hope in future you will have time to redesign your app.

**Comment by theAkito** [comment]
Greetings,

was reading this project's issues regarding pattern matching on websites and notifications regarding only based on pre-defined keywords and in the process stumbled upon this issue.

@rzafiamy

If you want to give my app a try, it focuses entirely on checking if a website is up or down, in various ways:

https://github.com/theAkito/webmon
